site/js/server.js

```diff
@@ -0,0 +1,53 @@
+var Server = {
+    Automaton: {
+        create: function( data, callback, error ) {
+            $.post( 'api/automaton/create', {
+                name: '',
+                data: data
+            }, function( result ) {
+                callback( result );
+            }, 'json' ).error( function( jqXHR, settings, thrownError ) {
+                error( thrownError );
+            } );
+        },
+        view: function( id, callback, error ) {
+            $.get( 'api/automaton/' + id, {}, function( result ) {
+                callback( result );
+            }, 'json' ).error( function( jqXHR, settings, thrownError ) {
+                error( thrownError );
+            } );
```

Comment:
You could construct the error-handling function with a factory so that you DRY:

```
function makeErrorHandler( error ) {
   return function( jqXHR, settings, thrownError ) {
      error( thrownError );
   };
}
```

Then just:

```
.error( makeErrorHandler( error ) )
```